Files/Interacts/Interaction.cs

```diff
@@ -495,6 +496,28 @@ public async Task<bool> OpenPath(string path, FilesystemItemType? itemType = nul
                 {
                     await InvokeWin32ComponentAsync(path);
                 }
+                else if (isShortcutItem)
```

Comment:
@gave92 this is what I have in mind

Reply:
Well isShortcutItem is nothing more than:
```cs
isShortcutItem  = itemPath.EndsWith(".lnk") || itemPath.EndsWith(".url")
```
The problem is that getting the TargetPath of a shortcut item requires calling the fulltrust process, if we can avoid doing that when we already know it (when we have a ListedItem) even better.